releasenotes/notes/49896.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 49896
+releaseNotes:
+- |
+  **Added** Allow user to define the traffic address type (service, workload, all or none) for their waypoint via the --for flag on the waypoint apply cmd. 
```

Comment:
```suggestion
  **Added** ability to define the traffic address type (service, workload, all or none) for waypoints via the `--for` flag when using `istioctl experimental waypoint apply` command.
```

Reply:
Sorry I missed this. Here's the follow-up PR: https://github.com/istio/istio/pull/49942

Thanks for the suggested improvement, it reads better!